src/locale/languages.ts

```diff
@@ -42,6 +43,7 @@ interface AppLanguageConfig {
 export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.en, name: 'English'},
   {code2: AppLanguage.an, name: 'Aragonés – Aragonese'},
+  {code3: AppLanguage.ast, name: 'Asturianu - Asturian'},
```

Comment:
How does this work? I don't think the types match up (currently `code2` is the required field on this type, and all the application logic also checks for `code2` here).

I think this needs a change in the application code to work.